internal/nostr/push.go

```diff
@@ -11,6 +17,57 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func (svc *Service) encryptToken(token string) (string, error) {
+	key := []byte(svc.Cfg.EncryptionKey)
+	block, err := aes.NewCipher(key)
+	if err != nil {
+		return "", err
+	}
+
+	gcm, err := cipher.NewGCM(block)
+	if err != nil {
+		return "", err
+	}
+
+	nonce := make([]byte, gcm.NonceSize())
+	if _, err := io.ReadFull(rand.Reader, nonce); err != nil {
+		return "", err
+	}
+
+	ciphertext := gcm.Seal(nonce, nonce, []byte(token), nil)
+	return base64.StdEncoding.EncodeToString(ciphertext), nil
```

Comment:
This is a bit different to the alby hub version e.g. how the nonce is stored. What is the benefit in your opinion?

Reply:
I think appending using "-" feels a bit hacky, and I saw other people using the same on GitHub so I followed this, wdyt?